### ISSUE TITLE
Duplicate *.block in Data/Sample/configure.ac

### ISSUE THREAD

**Issue by svigerske** [coin-or-tools/BuildTools#108]
Issue created by migration from Trac.

Original creator: @tkelman

Original creation time: 2013-11-09 03:38:00

Assignee: @svigerske

Version: 

Keywords: data sample

!Data/Sample doesn't seem to have its own project page, so I'll report this here.

https://projects.coin-or.org/svn/Data/Sample/stable/1.2/configure.ac has two copies of the line `AC_COIN_EXAMPLE_FILES([*.block])`

Revision 318 was committed to stable but not trunk, then 345 applied almost the same change to trunk, merged to stable in 346, but in a slightly different order resulting in the duplicate. This causes make install to fail with newer automake (I'm testing the autotools-update branch):
```
/usr/bin/install: will not overwrite just-created `</path/to/file>.block' with `<samefile>.block'
```

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-11-09 15:41:15

Thanks for catching this.
Fixed now in rev 401 in Data/stable/1.2.
Will do a new release soon.

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-11-09 15:41:15

Resolution: fixed
